skforecast/ForecasterAutoreg/ForecasterAutoreg.py

```diff
@@ -1102,23 +1102,24 @@ def _binning_in_sample_residuals(
 
     def _create_predict_inputs(
         self,
-        steps: int,
+        steps: Union[int, str, pd.Timestamp], 
         last_window: Optional[Union[pd.Series, pd.DataFrame]] = None,
         exog: Optional[Union[pd.Series, pd.DataFrame]] = None,
         predict_boot: bool = False,
         use_in_sample_residuals: bool = True,
         use_binned_residuals: bool = False,
-        check_inputs: bool = True
-    ) -> Tuple[np.ndarray, Optional[np.ndarray], pd.Index]:
+        check_inputs: bool = True,
+    ) -> Tuple[np.ndarray, Optional[np.ndarray], pd.Index, int]:
         """
         Create the inputs needed for the first iteration of the prediction 
         process. As this is a recursive process, the last window is updated at 
         each iteration of the prediction process.
         
         Parameters
         ----------
-        steps : int
-            Number of future steps predicted.
+        steps : Union[int, str, pd.Timestamp]
```

Comment:
Adapt to follow the same format as other arguments, "steps : int, str, pandas Timestamp"